src/pages/index.js

```diff
@@ -45,121 +43,157 @@ const SEO_TITLE = 'The easiest way to get Spotify Backstage';
-      icon: BeakerIcon,
-    }, {
-      name: 'Secure connections',
-      description: 'Securely connect to SaaS services and your own infrastructure using our tunneling broker service.',
-      icon: ShieldCheckIcon,
-    }],
-
-  }, {
-    title: 'Maintenance free',
-    description: 'Roadie handles upgrades and security so you can stay focused.',
-    illustration: {
-      webp: SecurityMaintenanceIllustration,
-      png: SecurityMaintenanceIllustrationPng,
-      alt: 'A progress bar and cog to indicate an upgrade in progress.',
-    },
-    bullets: [{
-      name: 'Automated upgrades',
+  features: [
+    {
+      title: 'Focus on building value instead of learning Backstage internals',
       description:
-        `Open-source means Backstage moves quickly. Upgrades can take days, and cause downtime when they go wrong. Why spend the time?`,
-      icon: ChipIcon,
-    }, {
-      name: 'Security patches delivered',
+        'Roadie manages deployments and security for you and helps you get the best out of Backstage with a few clicks.',
+      illustration: {
+        webp: DragDropIllustration,
+        png: DragDropIllustrationPng,
+        alt: 'A component from a Backstage plugin being dragged around with the mouse pointer',
+      },
+      bullets: [
+        {
+          name: 'Secure connections',
+          description:
+            'Securely connect to SaaS services and your own infrastructure using our tunneling broker service.',
+          icon: ShieldCheckIcon,
+        },
+        {
+          name: 'Custom plugin support',
+          description:
+            'Building your own internal plugins? Publish them to our private repository and they appear where you need them.',
+          icon: BeakerIcon,
+        },
+      ],
+    },
+    {
+      title: 'Keep your Backstage ever-green',
       description:
-        `We regularly patch vulnerabilities in the open-source code, ensuring you're using a hardened version of Backstage.`,
-      icon: ShieldCheckIcon,
-    }],
-  }],
+        'All Roadie customers are running on the latest Backstage version (1.3.0). Get the latest features from the community without the maintenance burden.',
+      illustration: {
+        webp: SecurityMaintenanceIllustration,
+        png: SecurityMaintenanceIllustrationPng,
+        alt: 'A progress bar and cog to indicate an upgrade in progress.',
+      },
+      bullets: [
+        {
+          name: 'Automated upgrades',
+          description: `Self-hosted Backstage teams can take up to 30% of their time upgrading their instance. We manage upgrades for you, whether they're security patches, minor, or major releases.`,
+          icon: ChipIcon,
+        },
+        {
+          name: 'SLAs Available',
+          description: `Downtime in Backstage can cause friction in your team and hinder adoption. We offer reliability and 24/7 support.`,
```

Comment:
Careful with "24/7 support". We have 24/7 on-call, but not support (ie. answering slack questions).

Reply:
Gotcha, how about "We offer reliability and 24/7 on-call engineers." Do we offer any concrete uptime?